test/test-clean.ts

```diff
@@ -49,6 +49,24 @@ describe('clean', () => {
     });
   });
 
+  it('should gracefully handle JSON with comments', () => {
```

Comment:
I think that these assertions should be async too? ` () => {` to `async () => {`

Reply:
Nope!  They return a promise, since `withFixtures` returns a promise, which seems to be what mocha is actually looking for.  All the tests in this file seem to work like that 🤷